packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### New features and enhancements
 
+- Updated `Gui.showQuickPick` to support selecting multiple `quickPickItem`s. [#3526](https://github.com/zowe/zowe-explorer-vscode/pull/3526)
```

Comment:
- We should explain what `Gui.showQuickPick` is: "Updated the `Gui.showQuickPick` **package/rocket ship/whatever** to support ... "
- We should avoid pluralizing  `quickPickItem` like this ^^: " ... selecting more than one  `quickPickItem`."